bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/scheduler/CronExpressionTest.java

```diff
@@ -63,4 +62,22 @@ public void getFinalTimeCheck() throws ParseException {
 
         assertEquals(checkDate, nextDate);
     }
+
+    @Test
+    public void runForever() throws ParseException, InterruptedException {
```

Comment:
You shouldn't name a test "run forever", this could be a little bit misleading :wink: